test/new-e2e/containers/ecs_test.go

```diff
@@ -0,0 +1,92 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+package containers
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/cenkalti/backoff"
+	"github.com/vboulineau/pulumi-definitions/aws"
+	"github.com/vboulineau/pulumi-definitions/aws/ecs/ecs"
+
+	"github.com/DataDog/datadog-agent/test/new-e2e/utils/credentials"
+	"github.com/DataDog/datadog-agent/test/new-e2e/utils/infra"
+
+	"github.com/pulumi/pulumi/sdk/v3/go/auto"
+	"github.com/pulumi/pulumi/sdk/v3/go/pulumi"
+	"github.com/stretchr/testify/require"
+
+	datadog "gopkg.in/zorkian/go-datadog-api.v2"
+)
+
+func TestAgentOnECS(t *testing.T) {
+	config := auto.ConfigMap{}
+	env := aws.NewSandboxEnvironment(config)
+	credentialsManager := credentials.NewManager()
+
+	apiKey, err := credentialsManager.GetCredential(credentials.AWSSSMStore, "agent.ci.dev.apikey")
+	require.NoError(t, err)
+	appKey, err := credentialsManager.GetCredential(credentials.AWSSSMStore, "agent.ci.dev.appkey")
+	require.NoError(t, err)
+
+	stack, err := infra.NewStack(context.Background(), "ci", "ecs-fargate", config, func(ctx *pulumi.Context) error {
+		return ecs.Run(ctx, env)
+	})
+	require.NoError(t, err)
+	require.NotNil(t, stack)
+
+	ctx, cancel := context.WithTimeout(context.Background(), 10*time.Minute)
+	result, err := stack.Up(ctx)
+	cancel()
+	require.NoError(t, err)
+
+	// ctx.Export("ecs-cluster-name", ecsCluster.Name)
+	// ctx.Export("ecs-cluster-arn", ecsCluster.Arn)
+	// ctx.Export("ecs-task-arn", taskDef.TaskDefinition.Arn())
```

Comment:
Task ARN might be interesting as it's the closest equivalent of "host" in ECS Fargate.